pkg/pipeline/pipeline_test.go

```diff
@@ -18,15 +18,18 @@
 package pipeline
 
 import (
-	"github.com/json-iterator/go"
+	"testing"
+
+	"github.com/sirupsen/logrus"
+
+	jsoniter "github.com/json-iterator/go"
 	"github.com/netobserv/flowlogs2metrics/pkg/config"
```

Comment:
goland editor will re-order these imports to be in alphabetical order.

Reply:
In `netobserv` we usually sort the imports with `goimports`, which is independent of the IDE (many team members use VScode). It can be configured to be used from Goland/IDEA too: 

<img width="655" alt="image" src="https://user-images.githubusercontent.com/939550/155965592-94b123b5-b51d-488d-9fd9-787c62700710.png">

But if this is an inconvenience for you I can adapt to use the default Goland. WDYT @jotak @jpinsonneau @OlivierCazade ?

Reply:
I'm also using `goimports` (configured in vscode)